release-notes/stable.yml

```diff
@@ -1,4 +1,12 @@
 releases:
+  40.20240920.3.0:
+    issues:
+      - text: "gcp: support c3 metal instance types"
+        url: "https://github.com/coreos/fedora-coreos-tracker/issues/1794"
```

Comment:
since a user [reported issues in the F40 version of this](https://github.com/coreos/fedora-coreos-tracker/issues/1797#issuecomment-2359114508) let's just wait to promote this one to "fixed" when we ship F41.

Reply:
I didn't see the F41 label on that issue, oops! I'll remove this entry.